packages/nuxt/src/module.ts

```diff
@@ -98,7 +99,10 @@ export default defineNuxtModule<ModuleOptions>({
 
     // Add auto-imports for Clerk components, composables and client utils
     addImportsDir(resolver.resolve('./runtime/composables'));
-    addImportsDir(resolver.resolve('./runtime/client'));
+    addImports({
+      name: 'createRouteMatcher',
+      from: resolver.resolve('./runtime/client'),
+    });
```

Comment:
Switched to using [`addImports`](https://nuxt.com/docs/api/kit/autoimports#addimports) to have more control over the auto-imported functions with a single point of import